python/paddle/trainer_config_helpers/layers.py

```diff
@@ -3334,6 +3335,60 @@ def block_expand_layer(input,
 
 
 @wrap_name_default()
+@layer_support()
+def maxout_layer(input,
+                 channels,
```

Comment:
channels=None,  channels can be got from the previous layer. So, better to automatically setting.